packages/camera/camera_android_camerax/lib/src/system_services.dart

```diff
@@ -0,0 +1,135 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:camera_platform_interface/camera_platform_interface.dart'
+    show CameraException, DeviceOrientationChangedEvent;
+import 'package:flutter/services.dart';
+
+import 'android_camera_camerax_flutter_api_impls.dart';
+import 'camerax_library.pigeon.dart';
+
+// ignore_for_file: avoid_classes_with_only_static_members
```

Comment:
I initially wanted to group all static calls to system services here, so I ignored this lint warning. Open to feedback on this decision.

Reply:
This is fine since you are just wrapping a native class. Can you also add a comment with this explanation about why it is ignored.